extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/user/CASAuthenticatedUser.java

```diff
@@ -50,6 +51,11 @@
      */
     private Map<String, String> tokens;
 
+    /**
+     * The unique identifiers of all user groups which a user is a member of.
```

Comment:
I think you mean "this user" here.

Reply:
Corrected. Yes, it should have been "this user"